sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LdeviceStatusEnum.java

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: 2021 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons.util;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+
+/**
+ * Object describing Ldevice Status, prefere this to LdeviceStatus constants
+ */
+public enum LdeviceStatusEnum {
```

Comment:
A class called [LDeviceStatus](/com-pas/compas-sct/blob/develop/sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LDeviceStatus.java#L12) already exists. You can use or delete it and keep the one you created.

Reply:
Removed